src/protocol.rs

```diff
@@ -202,7 +202,7 @@ impl<'a> Protocol<'a> {
             }
             "p2p-websocket-star" => Ok(Protocol::P2pWebSocketStar),
             "p2p-webrtc-star" => Ok(Protocol::P2pWebRtcStar),
-            "webrtc" => Ok(Protocol::WebRTC),
+            "webrtc-direct" => Ok(Protocol::WebRTCDirect),
```

Comment:
I must say having both `webrtc-direct` and `p2p-webrtc-direct` is confusing, but I guess nothing we can do about it other than good documentation.

Reply:
We could as well deprecate `p2p-webrtc-direct`. I am not aware of anyone using it. Though intuitively, that decision should happen on https://github.com/multiformats/multiaddr.